mailpoet/lib/Newsletter/Scheduler/Scheduler.php

```diff
@@ -81,4 +77,18 @@ public function getNewsletters(string $type, ?string $group = null): array {
   public function formatDatetimeString($datetimeString) {
     return Carbon::parse($datetimeString)->format('Y-m-d H:i:s');
   }
+
+  /**
+   * @return \DateTime|false
+   */
+  public function getNextRunDateTime($schedule, $fromTimestamp = false) {
```

Comment:
Is this change necessary? I see that you created it probably due to storing `scheduledAt`. But maybe it would be easier to use `Carbon::createFromFormat` because we know the format. What do you think? You could also avoid formatting the date into logs.

Reply:
I'd say this is necessary since I'd normally expect a higher lower level class to give me `raw`  information and not formatted one, and besides why should we format and create a string out of `DateTime` and then use the string to create `DateTime` again? It doesn't make much sense even though it's pretty easy to do so.